src/common/GurobiWrapper.h

```diff
@@ -71,6 +71,12 @@ class GurobiWrapper
     // Add a new EQ constraint, e.g. 3x + 4y = -5
     void addEqConstraint( const List<Term> &terms, double scalar );
 
+    // Add a piece-wise linear constraint
```

Comment:
Is this a new Gurobi interface we're starting to use? Perhaps all existing encodings should use it, too?

Reply:
I agree it's worth studying whether we should switch to gurobi PLW constraints altogether